testbed/core/models/actor.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+from django.apps import apps
+from datetime import timezone
+from ..utils.validators import validate_username
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+    
+class Actor(models.Model):
+    username = models.CharField(
+        max_length=100, unique=True, validators=[validate_username]
+    )
+    full_name = models.CharField(max_length=100)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    previously = models.JSONField(default=list, null=True, blank=True)
+
+    def __str__(self):
+        return self.username
+    
+    # Record a previous location of this account
+    def record_move(self, previous_server, previous_username, move_date=None):
+        print(f"Previously type: {type(self.previously)}")  # Debug line
```

Comment:
This was probably already here and just copied over, but as long as I'm noticing -- if we keep debug lines around we should move them to "log.debug" messages instead of prints.  There's lots of stuff you can do with log.debug messages that you can't do with prints.

Reply:
Oh, they are not meant to be there. I used them to fix a bug I had and I didn't delete them.